pkg/assembler/backends/testing/ingest_artifact.go

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
+
+import (
+	"strings"
+
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+)
+
+func registerAllArtifacts(client *demoClient) {
+	// strings.ToLower(string(checksum.Algorithm)) + ":" + checksum.Value
+	client.registerArtifact("sha256", "6bbb0da1891646e58eb3e6a63af3a6fc3c8eb5a0d44824cba581d2e14a0450cf")
+	client.registerArtifact("sha1", "7A8F47318E4676DACB0142AFA0B83029CD7BEFD9")
```

Comment:
still needs to be lowercase?

Reply:
they are lowercased in the `registerArtifact` below

Reply:
ah ok i was thinking that we should try to communicate that we want the input to be lowercase, but if we're going to have ingestion lower case it then we should be OK